docs/en/ingest-management/fleet/fleet-api-docs.asciidoc

```diff
@@ -1,10 +1,450 @@
+  --url 'https://my-kibana-host:9243/api/fleet/agent_policies?sys_monitoring=true' \
+  --header 'Accept: */*' \
+  --header 'Authorization: ApiKey yourbase64encodedkey' \
+  --header 'Cache-Control: no-cache' \
+  --header 'Connection: keep-alive' \
+  --header 'Content-Type: application/json' \
+  --header 'kbn-xsrf: xxx' \
+  --data '{
+  "name": "Agent policy 1",
+  "description": "",
+  "namespace": "default",
+  "monitoring_enabled": [
+    "logs",
+    "metrics"
+  ]
+}'
+----
+
+****
+beta[] To save time, you can use {kib} to generate the API request, then run it
+from the Dev Tools console. 
+
+. Go to **{fleet} -> Agent policies**.
+. Click **Create agent policy** and give the policy a name.
+. Click **Preview API request**.
+. Click **Open in Console** and run the request.
+
+****
+
+Example response:
+
+[source,shell]
+----
+{
+  "item": {
+    "id": "2b820230-4b54-11ed-b107-4bfe66d759e4", <1>
+    "name": "Agent policy 1",
+    "description": "",
+    "namespace": "default",
+    "monitoring_enabled": [
+      "logs",
+      "metrics"
+    ],
+    "status": "active",
+    "is_managed": false,
+    "revision": 1,
+    "updated_at": "2022-10-14T00:07:19.763Z",
+    "updated_by": "1282607447",
+    "schema_version": "1.0.0"
+  }
+}
+----
+<1> Make a note of the policy ID. You'll need the policy ID to add integration
+policies.
+
+[discrete]
+[[create-integration-policy-api]]
+== Create integration policy
+
+To create a integration policy (also known as an package policy) and add it an 
+existing agent policy, call `POST /api/fleet/package_policies`.
+
+This cURL example creates a integration policy for Nginx and adds it to the
+agent policy created in the previous example:
+
+[source,shell]
+----
+curl --request POST \
+  --url 'https://my-kibana-host:9243/api/fleet/package_policies' \
+  --header 'Authorization: ApiKey yourbase64encodedkey' \
+  --header 'Content-Type: application/json' \
+  --header 'kbn-xsrf: xx' \
+  --data '{
+  "name": "nginx-demo-123",
+  "policy_id": "2b820230-4b54-11ed-b107-4bfe66d759e4",
+  "package": {
+    "name": "nginx",
+    "version": "1.5.0"
+  },
+  "inputs": {
+    "nginx-logfile": {
+      "streams": {
+        "nginx.access": {
+          "vars": {
+            "tags": [
+              "test"
+            ]
+          }
+        },
+        "nginx.error": {
+          "vars": {
+            "tags": [
+              "test"
+            ]
+          }
+        }
+      }
+    }
+  }
+}'
+----
+
+****
+* beta[] To save time, you can use {kib} to generate the API call, then run it
+from the Dev Tools console. 
++
+. Go to **Integrations**, select an {agent} integration, and click
+**Add <Integration>**.
+. Configure the integration settings and select which agent policy to use.
+. Click **Preview API request**.
++
+If you're creating the integration policy for a new agent policy, the preview
+shows two requests: one to create the agent policy, and another to create the
+integration policy.
```

Comment:
Do you need a comma here? 

```suggestion
If you're creating the integration policy for a new agent policy, the preview
shows two requests: one to create the agent policy and another to create the
integration policy.
```

Reply:
Here I think the comma actually helps with readability, so I'm leaving it.